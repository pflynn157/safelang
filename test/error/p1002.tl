
func main is
    var x : i32 := 20;
    x := 30;
    
    x := true;
    x := false;
    
    x := "Hello!";
    x := 'A';
    
    x := true * 2;
end

