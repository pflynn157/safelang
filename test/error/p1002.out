P1_INCP_TYPE;
P1_INCP_TYPE;
P1_INCP_TYPE;
P1_INCP_TYPE;
P1_INCP_TYPE;
